src/custom-types.json

```diff
@@ -583,6 +583,13 @@
       "http://facebook.github.io/react/docs/getting-started.html"
     ]
   },
+  "text/vue": {
+    "compressible": true,
+    "extensions": ["vue"],
+    "sources": [
+      "http://vuejs.org/v2/guide/single-file-components.html"
```

Comment:
I'm wondering if that's really necessary? There's nothing about `text/jsx` on the page <http://facebook.github.io/react/docs/getting-started.html> either.